saleor/dashboard/widgets.py

```diff
@@ -15,3 +18,7 @@ def __init__(self, attrs=None):
         self.currency = getattr(settings, 'DEFAULT_CURRENCY')
         widgets = (PriceInput(self.currency), PriceInput(self.currency))
         super(RangeWidget, self).__init__(widgets, attrs)
+
+
+class PhonePrefixWidget(StorefrontPhonePrefixWidget):
+    template_name = 'dashboard/order/widget/phone-prefix-widget.html'
```

Comment:
We have recently settled on a naming convention for the template files to use underscores. Dashes should be used for static files that are accessible from an URL.

Reply:
Right!